.gitignore

```diff
@@ -34,3 +34,6 @@
 
 # Add back any keep files that might have been excluded by the above
 !.keep
+
+# SimpleCov
+coverage
```

Comment:
Small nitpick - can you change this to:

```
# Ignore coverage reports from SimpleCov
/coverage/*
```

and put it on line 9 before `# Ignore logfiles and tempfiles.` - the `!.keep` should be the last thing and file is sort of alphabetical

Reply:
@pixeltrix Sure thing. No problem.